config/300-clusterimagepolicy.yaml

```diff
@@ -482,6 +494,12 @@ spec:
                     fetchConfigFile:
                       description: 'FetchConfigFile controls whether ConfigFile will be fetched and made available for CIP level policy evaluation. Note that this only gets evaluated (and hence fetched) iff at least one authority matches. The ConfigFile will then be available in this format: https://github.com/opencontainers/image-spec/blob/main/config.md'
                       type: boolean
+                    includeObjectMeta:
```

Comment:
I start to wonder if it would be better to have a single parent field and under it the objectMeta and Spec.

Reply:
Yeah, hm. I don't really care one way or another :) Just one more level of indentation :) I don't see any other fields really coming here, we should pretty much have everything now, so guess if we change it to have include way you'd need to include 3 lines (include + children) to get all, where as now only 2.